packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -43,6 +60,45 @@ export default function SidebarNavigationScreenNavigationMenu() {
 
 	const menuTitle = navigationMenu?.title?.rendered || navigationMenu?.slug;
 
+	const handleSave = async () => {
+		saveEditedEntityRecord( 'postType', postType, postId );
+		setOpen( false );
+	};
+	const handleChange = ( title ) =>
+		editEntityRecord( 'postType', postType, postId, { title } );
+	const handleDelete = () =>
+		deleteEntityRecord( 'postType', postType, postId, { force: true } );
```

Comment:
what does `{ force: true }` do?

Reply:
It forces the deletion ignoring any constraints or dependencies. I used it because we are also using it [here](https://github.com/WordPress/gutenberg/blob/7180361aaa5470ba4987d80eed4d5fedc295130b/packages/block-library/src/navigation/edit/navigation-menu-delete-control.js#L67) when we delete menus from the inspector sidebar.